assembl/static2/js/app/components/debate/survey/posts.jsx

```diff
@@ -24,12 +24,15 @@ type Props = {
   posts: {
     edges: Array<PostNode>
   },
+  params: Object,
```

Comment:
`Object` is a generic type, can you be more specific here ?

Reply:
now i have created a RouterParams type, yes, i guess i can

Reply:
i encounter some issues to use the RouterParams type there, 
a a beginner in flow, i will ask for vincent'help when he comes back
for the moment, i merge the request

Reply:
(when @pascalhuynh will have reviewed my last commit)

Reply:
I just tested putting `params: RouterParams,` here. I have no errors.